src/Tizen.NUI/src/public/BaseComponents/TextMapHelper.cs

```diff
@@ -230,15 +230,14 @@ public static InputFilter GetInputFilterStruct(PropertyMap map)
         {
             string accepted = "";
             string rejected = "";
-            map.Find(0)?.Get(out accepted);
-            map.Find(1)?.Get(out rejected);
+            map.GetValue(0)?.Get(out accepted);
```

Comment:
Could you please share an example of the problem?

Reply:
Hello, 
Please refer the case: "TextMapHelperGGetInputFilterStruct" ,
and the path is: TizenFX\test\Tizen.NUI.Tests\Tizen.NUI.Devel.Tests\testcase\public\BaseComponents\TSTextMapHelper.cs

In this case, we will got failed, the out string is empty.